hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -366,51 +366,51 @@ object HoodieSparkSqlWriter {
       schema = HoodieAvroUtils.getNullSchema.toString
     }
 
-    // Handle various save modes
     if (mode == SaveMode.Ignore && tableExists) {
       log.warn(s"hoodie table at $basePath already exists. Ignoring & not performing actual writes.")
       false
```

Comment:
Wouldn't this method return false and exit from here if `mode == SaveMode.Ignore && tableExists`?

Reply:
It will return false and exit from here, and the same logic is used in `HoodieSparkSqlWriter.write`.But the original version will not directly return false, and will continue to execute the following code logic